app/utils/lava_log_parser.py

```diff
@@ -72,11 +76,52 @@ def run(log, boot, txt, html):
     else:
         boot_result_html = "<span class=\"warn\">{}</span>".format(boot_result)
 
+    # panic, error and warn are considered deprecated, but still here
```

Comment:
Is this comment still relevant?  I don't see "panic" in the list of formats.

Reply:
@gctucker updated